RPI/README.md

```diff
@@ -39,82 +39,71 @@ If the UART port is not set up correctly, there will be a file open error.
 
 **Install the packages necessary to run the demo.** 
 
-If you are cross-compiling, this would apply only to the compile host:
-```
-./mk/os-pkg/ubuntu-packages.sh
-sudo pip install -r  mk/python/pip_required_build.txt
-sudo pip install -r  mk/python/pip_required_gui.txt
-```
-For cross-compiling, clone the cross-compile tools from [here](https://github.com/raspberrypi/tools). This demo has configuration files that assume that the tools have been installed in `/opt/tools/rpi`. If they are installed elsewhere, the location can be updated [here](../mk/configs/host/Linux-generic.mk).
+Please see [INSTALL.md](../INSTALL.md) to ensure that the F´ application has been installed and tested with the basic Ref. For cross-compiling, clone the cross-compile tools from [here](https://github.com/raspberrypi/tools). This demo has configuration files that assume that the tools have been installed in `/opt/`. If they are installed elsewhere, the location can be updated [here](../cmake/toolchain/raspberrypi.cmake).
```

Comment:
@LeStarch Can you change the path to `/opt/tools/rpi`? That was the original location, because otherwise it is just `/opt/tools` which seems very vague.

Reply:
@timcanham this name comes from the name of the RPI GitHub checkout.  Their repository is somewhat ambiguously named "tools". Thus, we to have a location ending in "tools".  I wanted to make it simple:  clone directly into `/opt/`.  With the old location, I got really confused.  Was I cloning into /opt/tools/rpi or was in cloning into /opt/tools with a different repo name?

Would `/opt/rpi/tools` make the most sense?  Here the user creates `/opt/rpi` and clones into it.  Thoughts?

Reply:
Should be fixed to cloning in `/opt/rpi/`.